scripts/update-reference.js

```diff
@@ -86,17 +84,88 @@ async function copyExampleToRepo(exampleName) {
       process.exit();
     }
     const referencesCsv = fse.readFileSync(referencesCsvFile, 'UTF-8');
-    const exampleUrl = (
+    let exampleUrl = (
       (referencesCsv || '').split(/\r?\n/).find(s => s.startsWith('example,')) || ''
     ).split(',')[1];
+
+    // TODO: Deprecate this when all tests support the v2 format
+    // If v2 test format
+    if (exampleUrl === 'metadata')
+      exampleUrl = (
+        (referencesCsv || '').split(/\r?\n/).find(s => s.startsWith('example,')) || ''
+      ).split(',')[2];
```

Comment:
I know this pattern existed prior to this PR but wow. It is fine for now but we would thank ourselves later if we adopted a more spacious approach to things like this, perhaps write some helper functions

Reply:
I think this repo could benefit from more utility functions in general. I see `splitLines()`, `findLineswithPrefix()`, `extractNthValue()` here for example.

Reply:
@stalgiag agreed! I had the same thoughts while doing this but was more thinking of how it will look when v1 files no longer exist. In retrospect, that will take some time and it certainly helps to create that practice from now, for this file (and others).

I pushed a change for this in https://github.com/w3c/aria-at/pull/1076/commits/b3270285438a6ae8cb3b9b083c89d9463306d95c, let me know if this aligns with your thoughts.

Reply:
That is a major improvement. Thanks so much!